helm/examples/azure/sa/README.md

```diff
@@ -0,0 +1,70 @@
+## Directory contents
+
+### Introduction
+Persistent storage can be created manually or through Kubernetes Clusters. If created through Kubernetes the storage will be gone when Persistent Volume (PV) or Kubernetes clusters are deleted. For Azure manually created shared storage with a storage account will not depend on any Kubernetes cluster. The data generated will be persistent and re-used as long the PV/PVC names and mount points are the same.
+
+### Create a share with a storage account
+If you don't have a storage account and a shared volume you can create them. If you already have them, provide the information in env-sa and ../hpcc-azurefile/values.yaml and generate a Kuernetes secret, if  you do not have one yet."
+To create a shared storage including storage account you can reference Reference: https://docs.microsoft.com/en-us/azure/aks/azure-files-volume
+We provide a set of scripts to help with the process. Before you run any script make sure to login to Azure first:
+```console
+az login
+```
+
+### Environment variables
+env-sa defines some variables for creating a storage account and shared storage.
+Some variables have default values but some of them don't and you must provide the values:
+- SUBSCRIPTION: Azure subscription
+- STORAGE_ACCOUNT_NAME: Storage Account name which will be generated
+- SA_RESOURCE_GROUP: a resource group name which must be unique in your subscription
+- SA_LOCATION: the region the storage account will be created in. It is recommended to use the same region as the Kubernetes cluster.
+
+hpcc-azurefile/values.yaml defines tree variables needed for storage account and shared storage creation:
+- secretName:  a Kuberentes secret
+- secretNamespace: the namespace of the Kubernetes secret
+- shareName: shared storage name which will be used to create the storage
+
+### Create
+create-sa.sh will source env-sa, create a storage account and shared storage. It will also save the storage key to $SA_KEY_DIR/
+```console
+./create-sa.sh
+```
+### List
+list-sa.sh will list the shared storage
+```console
+./list-sa.sh
+```
+
+### Create a Kubernetes secret
+A kubernetes secret is required to use the storage. The secret name, secretName, is defined in hpcc-azurefile/values.yaml.<br/>
+If this Kuberenetes cluster has never generated the secret before, run following to generate the secret
+```console
+./create-secret.sh
+```
+### Install and use hpcc-azurefile
+The hpcc-azurefile helm chart will create the Persistent Volume and the Persistent Volume Claim for each of the required HPCC storage types based on the shared storage.
+Once installed the generated PVC names should be used when installing the HPCC helm chart.
+The examples/azure/values-retained-azurefile.yaml is an example of the HPCC storage settings that should be applied, after the "sa/hpcc-azurefile" helm chart is installed. "azurestorage-hpcc-azurefile" name is expected when launching "helm install" to match the PVC name in the values-retained-azurefile.yaml.
```

Comment:
looks like these 2 lines are a repeat of what's in the README.md in directory above, which is the right place to describe installing the hpcc-azurefile chart - should remove here.